packages/slate/test/models/text/marks/get-marks/marked-text.js

```diff
@@ -0,0 +1,18 @@
+/** @jsx h */
+
+import h from '../../../../helpers/h'
+import { Set } from 'immutable'
+import { Mark } from '../../../../../'
+
+export const input = (
+  <text>
+    <i>Cat is </i>
+    <b>Cute</b>
+  </text>
+)[0]
+
+export default function(t) {
```

Comment:
This linebreak before this export is good. All the other test files should be like this.